rllib/execution/rollout_ops.py

```diff
@@ -182,6 +192,9 @@ def __call__(self, batch: SampleBatchType) -> List[SampleBatchType]:
             self.count += batch.agent_steps()
 
         if self.count >= self.min_batch_size:
+            # BONSAI changes begin - added logging
+            logger.info(f"Completed to gather a full batch with size {self.count} with minimum batch size {self.min_batch_size}. Accumulated number of samples is {self.accumulated_count}.")
```

Comment:
my read is `self.accumulated_count` is always 0?

Reply:
Fixed.